.gitignore

```diff
@@ -6,3 +6,6 @@
 /pkg/
 /spec/reports/
 /tmp/
+/.idea
+/.DS_Store
```

Comment:
Do not add any platform-specific ignores (see: https://github.com/rails/rails/commit/5a8e7287c542cbe82d3127cae83dab8571ab9114).

Reply:
Thank you, I have created on my local machine .gitignore_global and reverted this commit.